modules/setting/audit.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2023 The Gitea Authors. All rights reserved.
```
Or even later, depending on when the PR is merged